### ISSUE TITLE
dRep 'choose' failing when using --SkipSecondary

### ISSUE THREAD

**Issue by alexmsalmeida** [MrOlm/drep#122]
Hi Matt,

I was trying to do a quick clustering with Mash for a set of genomes, but am getting a "KeyError" when skipping the secondary clustering. Find below the log:

```
Running pair-wise MASH clustering
  Will split genomes into 2 groups for primary clustering
2786 primary clusters made
Running secondary clustering
3. Nevermind! Skipping secondary clustering
Step 4. Return output
***************************************************
    ..:: dRep dereplicate Step 3. Choose ::..
***************************************************
    
Loading work directory
Traceback (most recent call last):
  File "/nfs/production/interpro/metagenomics/mags-scripts/miniconda3/lib/python3.7/site-packages/pandas/core/indexes/base.py", line 2646, in get_loc
    return self._engine.get_loc(key)
  File "pandas/_libs/index.pyx", line 111, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 138, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 1619, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 1627, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'reference'
```

I think this might be an issue specific to v3 (I am running v3.1.0) as I was previously able to do this with v2. Any ideas where the issue might be?

Many thanks in advance,
Alex

**Comment by MrOlm** [comment]
Hi Alex,

I'm 99% sure this is a bug that has been fixed in the lastest dRep version (3.2.1). Please let me know if it persists after updating.

Best,
Matt

**Comment by alexmsalmeida** [comment]
Perfect, thanks Matt. Can confirm issue has been solved in v3.2.1.

Thanks again,
Alex
